src/group_theory/torsion.lean

```diff
@@ -101,6 +129,27 @@ exponent_exists.is_torsion $ exponent_exists_iff_ne_zero.mpr exponent_ne_zero_of
 
 end group
 
+section module
+
+-- A (semi/)ring of scalars and a commutative monoid of elements
+variables (Fq R : Type*) [add_comm_monoid R]
```

Comment:
I think these variable names are better, especially since you don't assume finiteness in the first lemma. But also because there are finite rings that aren't fields. So the `Fq` seems to suggest the wrong thing, I think.
```suggestion
variables (R M : Type*) [add_comm_monoid M]
```